.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ defaults: &image
     - image: robojackets/roboracing-baseimage:latest
 
 defaults: &install_deps
-  run: apt-get update && rosdep update && rosdep install -iy --from-paths ./src --skip-keys='realsense2_camera pointgrey_camera_driver libuvc_camera'
+  run: apt-get update && rosdep update && rosdep install -iy --from-paths ./src --skip-keys='pointgrey_camera_driver libuvc_camera'
```

Comment:
Don't we have to submodule init and update when we test? I think we need to edit our checkout procedure in the Circle config

Reply:
Edit: this is fixed in the ebretl/136-vehicle-sim which branches off of this branch